pkg/upgrade/templates/v1/jiva_target.go

```diff
@@ -19,31 +19,32 @@ package templates
 var (
 	// JivaTargetPatch is generic template for target patch
 	JivaTargetPatch = `{
-		"metadata": {
-		   "labels": {
-			 "openebs.io/version": "{{.UpgradeVersion}}"
-		   }
-		},
-		"spec": {
-		   "template": {
-			  "metadata": {
-				 "labels":{
-					"openebs.io/version": "{{.UpgradeVersion}}"
-				 }
-			  },
-			 "spec": {
-			   "containers": [
-				 {
-					"name": "{{.ControllerContainerName}}",
-					"image": "{{.ControllerImage}}:{{.ImageTag}}"
-				 },
-				 {
-					"name": "maya-volume-exporter",
-					"image": "{{.MExporterImage}}:{{.ImageTag}}"
-				 }
-			   ]
-			 }
-		   }
-		}
-	  }`
+  "metadata": {
+    "labels": {
+      "openebs.io/version": "{{.UpgradeVersion}}"
+    }
+  },
+  "spec": {
+    "replicas": 1,
```

Comment:
[doubt] `replicas: 1` This is where we scale down the target right?

It would be better if the indentation changes were excluded, since it is generating a huge diff.

Reply:
Scale down is performed via update before patching. the replica deployment
This is where scale up the target back again while patching the new image and labels.